### PR TITLE
[3.6](Grid,TreeList,TreeView,Gantt,Scheduler): Drag and Drop document…

### PR DESCRIPTION
**Pull request by xristianstefanov**

### CODE REVIEW

File: components/grid/row-drag-drop.md [hunk 1]
```diff
@@ -54,7 +55,11 @@ The `OnRowDrop` event provides an object of type `GridRowDropEventArgs<T>` to it
     * `Below`
     * `Over`
     
-* `DestinationGrid` - the reference of the Grid in which the row is dropped. This is applicable when you drag and drop rows between different grids. 
+* `DestinationGrid` - the reference of the Grid in which the row is dropped. This is applicable when you drag and drop rows between different grids.
```
Comment (ntacheva) [inline components/grid/row-drag-drop.md:58]: Would be better to list the fields in a table.

File: components/grid/row-drag-drop.md [hunk 2]
```diff
@@ -238,6 +244,1450 @@ When you drap and drop items from one Grid to another, the `OnRowDrop` event fir
...
+public class BaseFlatItem
+{
+    public Guid Id { get; set; }
+    public Guid? ParentId { get; set; }
+    public string Text { get; set; }
+    public bool HasChildren { get; set; }
+}
+````
+
+>caption Drag and Drop between Grid and Gantt
```
Comment (ntacheva) [inline components/grid/row-drag-drop.md:993]:
```suggestion
```
If this example will be added later, we need to remove this title for now.

File: components/grid/overview.md [hunk 3]
```diff
@@ -175,6 +175,7 @@ The Grid has methods to execute actions such as:
 * [export to Excel]({%slug grid-export-excel%}) and other formats
 * [automatically resize columns to fit their content]({%slug components/grid/columns/resize%}#autofit-columns)
 * [get or set the Grid configuration state]({%slug grid-state%})
+* get the corresponding `TItem` of the destination Grid from the passed [`DestinationIndex`]({%slug grid-drag-drop-overview%}#event-arguments) - `GetItemFromDropIndex(string index)`
```
Comment (dimodi) [inline components/grid/overview.md:178]: This sentence is too technical and not understanble out of context. We can be more general -

```suggestion
* [get the dragged data item and its drop index from the destination Grid instance]({%slug grid-drag-drop-overview%})
```

File: components/scheduler/refresh-data.md [hunk 4]
```diff
@@ -13,9 +13,125 @@ position: 40
 @[template](/_contentTemplates/common/observable-data.md#intro)
 
 In this article:
+- [Rebind Method](#rebind-method)
 - [Observable Data](#observable-data)
 - [New Collection Reference](#new-collection-reference)
 
+## Rebind Method
+
+You can refresh the Scheduler data by using the Rebind method exposed to the reference of the TelerikScheduler.
```
Comment (ntacheva) [inline components/scheduler/refresh-data.md:22]: Some more info on when this is needed and when not. Also link to http://localhost/blazor-ui/common-features/data-binding/overview#refresh-data. You may use the template similar to https://github.com/telerik/blazor-docs/pull/1129/files#diff-8e3e1b52deeaa116313bff9e1941eb5df7ed9a71c14e75e2fb5d67d2b7e14cffR22

File: components/treelist/overview.md [hunk 5]
```diff
@@ -160,6 +160,7 @@ The TreeList component has methods to to execute actions such as:
 * [Rebind to refresh the data]({%slug treelist-refresh-data%}#rebind-method)
 * [Automatically resize columns to fit their content]({%slug treelist-columns-resize%})
 * [Get or set the TreeList configuration state]({%slug treelist-state%})
+* get the corresponding `TItem` of the destination TreeList from the passed [`DestinationIndex`]({%slug treelist-drag-drop-overview%}#event-arguments) - `GetItemFromDropIndex(string index)`
```
Comment (dimodi) [inline components/treelist/overview.md:163]:
```suggestion
* [Get the dragged data item and its drop index from the destination TreeList]({%slug treelist-drag-drop-overview%})
```

File: components/treelist/row-drag-drop.md [hunk 6]
```diff
@@ -51,7 +52,11 @@ The `OnRowDrop` event provides an object of type `TreeListRowDropEventArgs<T>` t
     * `Below`
     * `Over`
     
-* `DestinationTreeList` - the reference of the TreeList in which the row is dropped. This is applicable when you drag and drop rows between different TreeLists. 
+* `DestinationTreeList` - the reference of the TreeList in which the row is dropped. This is applicable when you drag and drop rows between different TreeLists.
```
Comment (ntacheva) [inline components/treelist/row-drag-drop.md:55]: Same suggestion as for the Grid - fields in a table.

File: components/grid/row-drag-drop.md [hunk 7]
```diff
@@ -238,6 +244,1450 @@ When you drap and drop items from one Grid to another, the `OnRowDrop` event fir
 }
 ````
 
+### Drag and Drop between Grid, TreeList, TreeView, Gantt and Scheduler
+
+The functionality allows dragging items between Grid, [TreeList]({%slug treelist-drag-drop-overview%}), [TreeView]({%slug treeview-drag-drop-overview%}), [Gantt]({%slug gantt-overview%}), and [Scheduler]({%slug scheduler-overview%}). To achieve it, set the `Draggable`/`RowDraggable` parameter, and implemented through an event -  `OnDrop`/`OnRowDrop`.
```
Comment (ntacheva) [inline components/grid/row-drag-drop.md:249]:
```suggestion
The functionality allows dragging items between Grid, [TreeList]({%slug treelist-drag-drop-overview%}), [TreeView]({%slug treeview-drag-drop-overview%}), [Gantt]({%slug gantt-overview%}), and [Scheduler]({%slug scheduler-overview%}). To achieve it, set the `Draggable`/`RowDraggable` parameter, and implement it through an event -  `OnDrop`/`OnRowDrop`.
```

File: components/treelist/row-drag-drop.md [hunk 8]
```diff
@@ -588,6 +594,352 @@ public class Employee
 }
 ````
 
+### Drag and Drop between TreeList, Grid, TreeView, Gantt and Scheduler
+
+The functionality allows dragging items between TreeList, [Grid]({%slug grid-drag-drop-overview%}), [TreeView]({%slug treeview-drag-drop-overview%}), [Gantt]({%slug gantt-overview%}), and [Scheduler]({%slug scheduler-overview%}). To achieve it, set the `Draggable`/`RowDraggable` parameter, and implemented through an event -  `OnDrop`/`OnRowDrop`.
```
Comment (ntacheva) [inline components/treelist/row-drag-drop.md:599]:
```suggestion
The functionality allows dragging items between TreeList, [Grid]({%slug grid-drag-drop-overview%}), [TreeView]({%slug treeview-drag-drop-overview%}), [Gantt]({%slug gantt-overview%}), and [Scheduler]({%slug scheduler-overview%}). To achieve it, set the `Draggable`/`RowDraggable` parameter, and implement it through an event -  `OnDrop`/`OnRowDrop`.
```

File: components/treeview/drag-drop.md [hunk 9]
```diff
@@ -47,14 +48,429 @@ The `OnDrop` event provides an object of type `TreeViewDropEventArgs` to its eve
     * `Below`
     * `Over`
     
-* `DestinationTreeView` - the reference of the TreeView in which the node is dropped. This is applicable when you drag and drop nodes between different instances of the component. 
+* `DestinationTreeView` - the reference of the TreeView in which the node is dropped. This is applicable when you drag and drop nodes between different instances of the component.
```
Comment (ntacheva) [inline components/treeview/drag-drop.md:51]: Same here - fields in table.

**Review by dimodi — APPROVED** [review]

**Pull request merged**